content/troubleshooting/scaledobject-not-scale.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "Why is the `ScaledObject` paused?"
+weight = 1
++++
+
+We discovered a bug in how KEDA manages the upstream errors. By design we'd like to keep the things like they are in case of failure getting the metrics from the upstream and the `fallback` section is not defined.
+
+Before v2.6.1 the behavior in that case was scaling to zero the workload (which is risky) and [we have fixed](https://github.com/kedacore/keda/pull/2604) it on v2.6.1 so after it, the new behavior is keeping the things like they are till having more information. This behavior could seem that the `ScaledObject` is paused (nothing will be done). 
```

Comment:
Should we even bother covering the behavior before v2.6.1? Doesn't it make it a bit more confusing?

Alternative: Cover it, but only in a bulb after the new behavior was explained similar to this:

```markdown
> ⚠️ Are you using KEDA v2.6.0 or below? Then your workload will be scaled to 0. Because this can be tricky, we have changed this behavior to be more safe and not impact production workloads.
```